### PR TITLE
PAT-1751 Next review date change fails for offender without a category

### PR DESCRIPTION
**Pull request by steverendell**

### CODE REVIEW

File: docker-compose-test.yml [hunk 1]
```diff
@@ -13,25 +13,24 @@ services:
...
-  localstack:
+  categorisation-localstack:
     image: localstack/localstack:0.11.2
     ports:
       - "4567-4584:4567-4584"
-      - "${PORT_WEB_UI-8088}:${PORT_WEB_UI-8088}"
     networks:
       - hmpps
-    container_name: localstack
+    container_name: categorisation-localstack
```
Comment (steverendell) [inline docker-compose-test.yml:33]: This enables having several docker compose's at once

File: integration-tests/src/test/groovy/uk/gov/justice/digital/hmpps/cattool/specs/LandingPageSpecification.groovy [hunk 2]
```diff
@@ -24,7 +24,6 @@ class LandingPageSpecification extends GebReportingSpec {
   def setup() {
     db.clearDb()
     elite2Api.stubAgencyDetails('LPI')
-    elite2Api.stubAssessments('B2345YZ')
   }
```
Comment (steverendell) [inline integration-tests/src/test/groovy/uk/gov/justice/digital/hmpps/cattool/specs/LandingPageSpecification.groovy:27]: moving stubAssessments() back into the individual tests wasnt actually needed in the end but i left it in as it more accurately emulates scenarios where there is no existing cat

File: test/routes/form.test.js [hunk 3]
```diff
@@ -649,8 +649,9 @@ describe('GET /ratings/violence', () => {
       .expect('Content-Type', /html/)
       .expect(res => {
         expect(res.text).toMatch(/Home.+Categorisation home.+Safety and good order/s)
-        expect(res.text).toContain(`This person has been reported as the perpetrator in 5 assaults in custody before,
-      including 2 serious assaults and 3 non-serious assaults in the past 12 months.`)
+        expect(res.text).toMatch(
+          /This person has been reported as the perpetrator in 5 assaults in custody before,\s+including 2 serious assaults and 3 non-serious assaults in the past 12 months./
+        )
```
Comment (steverendell) [inline test/routes/form.test.js:654]: these changes were needed to get tests working on windows (where there is a CR in the string)

File: server/views/pages/categoriserLanding.html [hunk 4]
```diff
@@ -77,7 +77,7 @@ <h1 class="govuk-heading-m">Start an initial categorisation</h1>
     {% endif %}
   </div>
 
-  {% if not inProgress(data) %}
+  {% if data.nextReviewDate and not inProgress(data) %}
```
Comment (steverendell) [inline server/views/pages/categoriserLanding.html:80]: " if it has a next review date then it can be changed"

**Review by connormaglynn — APPROVED** [review]
lgtm

**Pull request merged**